### DISCUSSION TITLE
New random multilevel sensor or meter entities appearing all the time

### DISCUSSION THREAD

**Discussion by jhein05** [zwave-js/zwave-js#3293] [Q&A]
Hi,

I have random multilevel sensors appearing all over the place. Here is an example:
![image](https://user-images.githubusercontent.com/5014124/132269410-147f524a-4131-44bf-9756-1fa73447ef0b.png)
This is an Aeotec Multisensor6, but it also happens with other devices like Aeotec, Inovelli or Zooz switches. I have 101 nodes all together and it seems completely random, but every day 3 to 5 nodes have new multilevel sensor or meter entities. They disappear after a re-interview.
I am using the zwavejs2mqtt addon on HA Blue hardware. Everything is on the newest version, but this has been happening for month and with all the versions that were released during that time frame. 

Any ideas why this might be happening?

**Comment by AlCalzone** [comment]
I've seen that a couple of times with multisensors. Apparently, every now and then one of the reports gets corrupted on the air, but in a way that the checksum still matches. Then you end up with a report that is completely nonsensical but still decodes. The problem is that older CC versions don't allow us to check which sensors should be there, so we kinda accept all valid reports.

We should probably check for the newer versions that the report matches the types that were indicated as supported.

**Reply by jhein05** [reply]
I am not 100% sure what that means :) Let me know if I can help debug in any way, please.

Just as a side note: I have 16 Multisensor 6 and it has happened with one since I made the original post. Usually 2-3 sensors have those random entities every 24h. I also don't recall that this ever happened with the old OpenZwave integration. It had a million other issues, but not this one :)

**Reply by sotatech** [reply]
Yeah, this happened a lot with the old OZW integration with Aeotec 4 in 1 multisensors. It would be nice to be able to lock the devices to the pre-defined sensors to avoid this.

**Comment by AlCalzone** [comment]
fixed in v8.5.0
